CHANGELOG.md

```diff
@@ -7,15 +7,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
----
-
 ### Added
 
 - Support for opentelemetry 0.21
+- `tracing_opentelemetry_0_21` and `tracing_opentelemetry_0_21` features
```

Comment:
I don't understand the difference :thinking:

Reply:
Oops my bad, fixed